src/api/loader.coffee

```diff
@@ -58,10 +58,12 @@ handleAPIEvent = (apiEventChannel, baseUrl, setting, requestEvent = {}) ->
     $.ajax(apiSetting)
       .then((responseData) ->
         delete LOADING[apiSetting.url]
-        completedEvent = interpolate(setting.completedEvent, requestEvent.data)
-        completedData = getResponseDataByEnv(isLocal, requestEvent, responseData)
-        apiEventChannel.emit(completedEvent, completedData)
-
+        try
```

Comment:
Hmm, ok. The "Caught Errors are normal" bit bugs me a lot (so axiom catching the errors silently is not a "feature" from my perspective :smile: ) but as it stands this PR seems to "catch any errors when getting a response and display an error message" rather than the misleading "Check for null response in ..."